openthread/include/openthread/thread_ftd.h

```diff
@@ -183,7 +183,7 @@ otError otThreadSetRouterEligible(otInstance *aInstance, bool aEligible);
  * Set the preferred Router Id.
  *
  * Upon becoming a router/leader the node attempts to use this Router Id. If the preferred Router Id is not set or if
- * it can not be used, a randomly generated router id is picked. This property can be set only when the device role is
+ * it cannot be used, a randomly generated router id is picked. This property can be set only when the device role is
```

Comment:
```suggestion
 * it cannot be used, a randomly generated Router ID is picked. This property can be set only when the device role is
```
Not sure whether router needs to be capitalized, but it was in other instances.